src/machines/modelingMachine.ts

```diff
@@ -2475,7 +2490,21 @@ export const modelingMachine = setup({
 
         // Extract inputs
         const ast = kclManager.ast
-        const { selection, length } = input
+        const { nodeToEdit, selection, length } = input
+
+        // If this is an edit flow, first we're going to remove the old node
+        const pipeIndex = 5
```

Comment:
Same as above `const pipeIndex = nodeToEdit.findIndex(([_, type]) => type === 'PipeExpression')`

Reply:
Changed, thank you!